symphony-bdk-core/src/main/java/com/symphony/bdk/core/ServiceFactory.java

```diff
@@ -68,13 +71,19 @@
   private final RetryWithRecoveryBuilder<?> retryBuilder;
 
   public ServiceFactory(ApiClientFactory apiClientFactory, AuthSession authSession, BdkConfig config) {
+    this.config = config;
     this.podClient = apiClientFactory.getPodClient();
     this.agentClient = apiClientFactory.getAgentClient();
     this.datafeedAgentClient = apiClientFactory.getDatafeedAgentClient();
     this.authSession = authSession;
     this.templateEngine = TemplateEngine.getDefaultImplementation();
-    this.config = config;
     this.retryBuilder = new RetryWithRecoveryBuilder<>().retryConfig(config.getRetry());
+
+    if (this.config.getCommonJwt().getEnabled()) {
+      final OAuthSession oAuthSession = new OAuthSession(authSession);
+      this.podClient.getAuthentications().put("bearerAuth", new OAuthentication(oAuthSession::getBearerToken));
```

Comment:
Maybe put it in a static final string.

Reply:
done https://github.com/finos/symphony-bdk-java/pull/618/commits/c7cd1cfe370a57d178dab78a22de227d5e199292